src/commands/link.ts

```diff
@@ -3,30 +3,32 @@ import { flags as oclifFlags } from '@oclif/command'
 import { CustomCommand } from '../api/oclif/CustomCommand'
 import { appLink } from '../modules/apps/link'
 
+import { ColorifyConstants } from '../api/constants/Colors'
+
 export default class Link extends CustomCommand {
-  static description = 'Start a development session for this app'
+  static description = `Locally syncs the app in the current directory with the ${ColorifyConstants.ID('development workspace')} in use.`
```

Comment:
I don't understand the `locally` in the description. The link happens in the cloud, everything workspace related happens in the cloud as far as I know

Reply:
Perhaps I just mismatched the command. I understood that it's related to the `vtex link` command

Reply:
I thought about local development. But yes, it's a bit confusing.  
What do you think about: 
Syncs the app in the current directory with the ${ColorifyConstants.ID('development workspace')} in use.
?

Reply:
Good Idea